doc/source/getting_started/running_mechanical.rst

```diff
@@ -49,52 +49,55 @@ Launch Mechanical locally with this code:
-Launch a Mechanical server from the command line
+Launch Mechanical from the command line
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-You can launch Mechanical from the command line in server mode and then
-manually connect to it.
-
-**On Windows**
-
-Assume that Mechanical is installed at ``C:/Program Files/ANSYS Inc/vXXX``
-, where ``XXX`` is the three-digit format for the version. For example,
-the path for 2023 R1 is typically ``C:/Program Files/ANSYS Inc/v231``.
-
-Launch Mechanical in server mode with this code:
-
-.. code::
+The `ansys-mechanical` utility is installed automatically with PyMechanical,
+and can be used to run Mechanical from the command line. To obtain help on
+usage, type this command:
 
-    C:/Program Files/ANSYS Inc/v231/aisol/bin/winx64/AnsysWBU.exe -DSApplet -AppModeMech -nosplash -notabctrl -grpc 10000
+.. code:: console
 
-**On Linux**
+    $ ansys-mechanical --help
 
-Assume that Mechanical 2023 R1 is installed at ``/usr/ansys_inc``.
+    Usage: ansys-mechanical [OPTIONS]
 
-Launch Mechanical in server mode with this code:
+        CLI tool to run mechanical.
 
-.. code::
+        USAGE:
 
-    /usr/ansys_inc/v231/aisol/.workbench -dsapplet -AppModeMech -nosplash -notabctrl -grpc 10000
+        This example demonstrates the main use of this tool:
 
+            $ ansys-mechanical -r 232 -g
 
-View server information
-~~~~~~~~~~~~~~~~~~~~~~~~
-As Mechanical starts in server mode, you can see the server information:
+            Starting Ansys Mechanical version 2023R2 in graphical mode...
 
-.. code::
+    Options:
+        -h, --help                 Show this message and exit.
+        -p, --project-file TEXT    Mechanical project file (.mechdb) to open. Cannot
+                                    be mixed with -i
+        --port INTEGER             Start mechanical in server mode with the given
+                                    port number
+        -i, --input-script TEXT    Name of the input Python script. Cannot be mixed
+                                    with -p
+        -s, --show-welcome-screen  Show the welcome screen, where you can select the
+                                    file to open.        Only affects graphical mode
+        --debug                    Show a debug dialog right when the process
+                                    starts.
+        -r, --revision INTEGER     Ansys Revision number, e.g. "241" or "232". If
+                                    none is specified, uses the default from ansys-
+                                    tools-path
+        -g, --graphical            Graphical mode
 
-    Starting the grpc server at port 10000
-    Started the grpc server at port 10000
+    ...
 
-If you want to configure the port that the Mechanical server listens on, when you launch
-Mechanical, use the ``-grpc`` argument. For example, on Linux, launch Mechanical 2023 R1
-on port ``10001`` with:
 
-.. code::
+You can launch Mechanical from the command line in server mode and then
```

Comment:
```suggestion
In server mode, you can launch Mechanical from the command line and then
```

Reply:
@sbradrick I reworded this, but made a slightly different choice on the sentence structure